### PR TITLE
aruha-984: create first timeline for event type by default

### PR DESCRIPTION
**Pull request by adyach**
[aruha-984: Create first timeline for event type by default](https://techjira.zalando.net/browse/ARUHA-984)

## Description
Remove topic from event type + remove fake timeline

## Review
- [ ] Tests
- [ ] Documentation
- [ ] CHANGELOG

## Deployment Notes
Block creation of event types during rollout

### CODE REVIEW

File: src/acceptance-test/java/org/zalando/nakadi/webservice/BlockEventPublishingAT.java [hunk 1]
```diff
@@ -25,7 +25,12 @@
     @Before
     public void setUp() throws JsonProcessingException {
         eventType = EventTypeTestBuilder.builder().build();
-        NakadiTestUtils.createEventTypeInNakadi(eventType);
+//        NakadiTestUtils.createEventTypeInNakadi(eventType);
```
Comment (antban) [inline src/acceptance-test/java/org/zalando/nakadi/webservice/BlockEventPublishingAT.java]: ?

File: src/main/java/org/zalando/nakadi/controller/EventTypeController.java [hunk 2]
```diff
@@ -210,4 +191,32 @@ private void setDefaultEventTypeOptions(final EventTypeBase eventType) {
...
+//        LOG.debug(exception.getMessage(), exception);
+//        return Responses.create(exception.asProblem(), request);
+//    }
+//
+//    @ExceptionHandler(InvalidEventTypeException.class)
+//    public ResponseEntity<Problem> invalidEventTypeException(final InvalidEventTypeException exception,
+//                                                             final NativeWebRequest request) {
+//        LOG.debug(exception.getMessage(), exception);
+//        return Responses.create(exception.asProblem(), request);
+//    }
```
Comment (antban) [inline src/main/java/org/zalando/nakadi/controller/EventTypeController.java]: ??? What is the problem

File: src/main/java/org/zalando/nakadi/controller/TimelinesController.java [hunk 3]
```diff
@@ -53,13 +53,6 @@ public TimelinesController(final TimelineService timelineService) {
         return ResponseEntity.status(HttpStatus.CREATED).build();
     }
 
-    @RequestMapping(value = "{id}", method = RequestMethod.DELETE)
-    public ResponseEntity<?> deleteTimeline(@PathVariable("name") final String eventTypeName,
-                                            @PathVariable("id") final String timelineId) {
-        timelineService.delete(eventTypeName, timelineId);
-        return ResponseEntity.ok().build();
-    }
-
```
Comment (antban) [inline src/main/java/org/zalando/nakadi/controller/TimelinesController.java]: The method information is not removed from documentation

File: src/main/java/org/zalando/nakadi/domain/EventTypeBase.java [hunk 4]
```diff
@@ -23,9 +23,6 @@
     @Size(min = 1, max = 255, message = "the length of the name must be >= 1 and <= 255")
     private String name;
 
-    @JsonIgnore
-    private String topic;
-
```
Comment (antban) [inline src/main/java/org/zalando/nakadi/domain/EventTypeBase.java]: Yeah!

File: src/main/java/org/zalando/nakadi/util/FeatureToggleServiceDefault.java [hunk 5]
```diff
@@ -12,7 +12,8 @@
 public class FeatureToggleServiceDefault implements FeatureToggleService {
     private static final Set<Feature> DEPRECATED_FEATURES = ImmutableSet.of(
             Feature.DISABLE_EVENT_TYPE_CREATION,
-            Feature.DISABLE_SUBSCRIPTION_CREATION
+            Feature.DISABLE_SUBSCRIPTION_CREATION,
+            Feature.CONNECTION_CLOSE_CRUTCH
```
Comment (antban) [inline src/main/java/org/zalando/nakadi/util/FeatureToggleServiceDefault.java]: Why?

**Comment by antban** [comment]
@adyach Finished review

File: src/acceptance-test/java/org/zalando/nakadi/webservice/BlockEventPublishingAT.java (hunk 1 above)
Comment (adyach) [reply src/acceptance-test/java/org/zalando/nakadi/webservice/BlockEventPublishingAT.java]: testing things

File: src/main/java/org/zalando/nakadi/controller/EventTypeController.java (hunk 2 above)
Comment (adyach) [reply src/main/java/org/zalando/nakadi/controller/EventTypeController.java]: testing things

File: src/main/java/org/zalando/nakadi/util/FeatureToggleServiceDefault.java (hunk 5 above)
Comment (adyach) [reply src/main/java/org/zalando/nakadi/util/FeatureToggleServiceDefault.java]: On mac it does not work other way

**Comment by adyach** [comment]
👍

**Comment by adyach** [comment]
deploy validation please

**Comment by adyach** [comment]
deploy validation please

**Pull request merged**